index.html

```diff
@@ -63,6 +63,199 @@ <h6>LET'S CONNECT</h6>
+          />
+          <div class="desktop-work-detail">
+            <h3>Multi-Post Stories</h3>
+            <ul class="project-details">
+              <li>CANOPY</li>
+              <li>BACKEND DEV</li>
+              <li>2015</li>
+            </ul>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="project_coding_langs">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="preview_button outlinings">See Project</button>
+          </div>
+        </article>
+        </article>
+      </div>
+    </section>
+
+    <section class="about-white-back">
+      <div class="desktop-about">
+        <h3>
+          About <br />
+          Myself
+        </h3>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="socials-about">
+          <h6>LET'S CONNECT</h6>
+          <ul class="social-icon">
+            <li>
+              <a href="#"><img src="./images/twitter.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/Linkedin.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/medium.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/github.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/victory.png" alt=""></a>
+            </li>
+          </ul>
+        </div>
+        <button class="outlinings">Get my resume</button>
+      </div>
+      <div class="desktop-lang">
+        <ul class="about__stacks">
+          <li class="about_stacks__item" style="border-bottom: 0px">
+            <span>Languages</span>
+            <i class="fa fa-angle-down"></i>
+          </li>
+          <ul class="lang">
+            <li class="js">
+              <img src="images/js.png" alt="" />
+              <h3>JavaScript</h3>
+            </li>
+            <li class="html">
+              <img src="images/html.png" alt="" />
+              <h3>HTML</h3>
+            </li>
+            <li class="css">
+              <img src="images/css.png" alt="" />
+              <h3>CSS</h3>
+            </li>
+          </ul>
+          <ul>
+          <li class="about_stacks__item">
+            <span>Frameworks</span>
+            <i class="fa-solid fa-chevron-right"></i>
+          </li>
+          <li class="about_stacks__item">
+            <span>Skills</span>
+            <i class="fa-solid fa-chevron-right"></i>
+          </li>
+        </ul>
+      </div>
+    </section>
+    
+    </section>
```

Comment:
- [ ] This closing tag does not close any section. Please remove it.
- [ ] Kindly add only one `section` element to wrap the whole of the Works section. This tag should be used to wrap sections only. You can replace it with `div` to wrap up each Work project.
- [ ] Please remember that after wrapping the Works section with the `section` element, you should use `display: grid` style to lay out its children. You should then use the Flex Box to lay out each child's content.
- [ ] Making this change will help you implement the change I mentioned in a comment above, about the margins and padding.